docs/source/index.rst

```diff
@@ -2,50 +2,50 @@ Welcome to daskperiment's documentation!
 ========================================
 
 `daskperiment` is a tool to perform reproducible machine learning experiment.
-It allows users to define and manage the history of trials
+It enables users to define and manage the history of trials
 (given parameters, results and execution environment).
 
-The package is built on `Dask`, a package for parallel computing with task
+This package is built on `Dask`, a package for parallel computing with task
 scheduling. Each experiment trial is internally expressed as `Dask` computation
 graph, and can be executed in parallel.
 
-It can be used in both Jupyter and command line (and also in standard Python interpreter). The benefits of daskperiemnt are:
+It can be used both on Jupyter and command line (and also on standard Python interpreter). The benefits of daskperiemnt are:
 
-- Usable in standard Python/Jupyter environment (and optionally with standard KVS).
+- Compatibility with standard Python/Jupyter environment (and optionally with standard KVS).
 
-  - No need to setting up server applications.
-  - No registration to cloud services.
-  - Not to be constrained by slightly customized Python shells.
+  - No need to set up server applications
+  - No need to registrate on any cloud services
+  - Run on your customized Python shells
```

Comment:
I meant it is usable in standard python environment (not locked in to 3rd vendor's impl).

Reply:
Modified the sentence and merged. Let me know if additional modification is required.